sdf/sdf3.go

```diff
@@ -1038,7 +1038,7 @@ type OffsetSDF3 struct {
 	bb     Box3    // bounding box
 }
 
-// Elongate3D returns the elongation of an SDF3.
+// Offset3D enlarges/shrinks an SDF uniformly outwards/inwards from all surfaces
```

Comment:
It's a bit non-intuitive. Consider a rectangle - An outside offset has rounded corners. An inside offset is rectangular.

Reply:
Definitely. Other operations also have non-intuitive behavior, like `Elongate3D` after `Translate3D` doesn't actually elongate the object you're modeling as you might think but spreads that elongation between the shape and the space from origin. Would be great to compile a list of gotchas like the above!